packages/analytics-js/src/services/StoreManager/StoreManager.ts

```diff
@@ -122,16 +126,18 @@ class StoreManager implements IStoreManager {
       'initialReferringDomain',
       'sessionInfo',
     ];
+
+    let storageEntries = {};
     userSessionKeyValues.forEach(sessionKey => {
       const key = sessionKey;
       const storageKey = sessionKey;
-      const providedStorageType = entries?.[key]?.type;
+      const configuredStorageType = entries?.[key]?.type;
 
       const preConsentStorageType = getStorageTypeFromPreConsent(state, sessionKey);
 
       // Storage type precedence order: pre-consent strategy > entry type > global type > default
       const storageType =
-        preConsentStorageType ?? providedStorageType ?? globalStorageType ?? DEFAULT_STORAGE_TYPE;
+        preConsentStorageType ?? configuredStorageType ?? globalStorageType ?? DEFAULT_STORAGE_TYPE;
```

Comment:
What will happen to `preConsentStorageType` after consent is given?
If it is still there then `configuredStorageType` will not affect the type.

Reply:
Done.